portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,21 +38,52 @@
 
 
   @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+   Query query = new Query("Task").addSort("text", SortDirection.DESCENDING);
+   DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();    
+
+   PreparedQuery results = datastore.prepare(query);
+   
+   List<Task> task = new ArrayList<Task>();
```

Comment:
nit: name this "tasks" or "taskList" so it's clear it can be multiple, and rename "task2" something like "task".

I was confused by why there was a second task until I realized the first task is actually a list, and those renames should make it clearer.